docs/_interface/Command-Line-Options.md

```diff
@@ -47,7 +47,7 @@ ETHEREUM OPTIONS:
   --keystore value                    Directory for the keystore (default = inside the datadir)
   --usb                               Enable monitoring and management of USB hardware wallets
   --pcscdpath value                   Path to the smartcard daemon (pcscd) socket file
-  --networkid value                   Explicitly set network id (integer)(For testnets: use --ropsten, --rinkeby, --goerli instead) (default: 1)
+  --networkid value                   Explicitly set network id (integer)(For testnets: --sepolia, --goerli instead) (default: 1)
```

Comment:
This file is the output of `geth --help`, no point in changing it. It will eventually follow the updates we make to the CLI.

Reply:
ok!

```suggestion
  --networkid value                   Explicitly set network id (integer)(For testnets: use --ropsten, --rinkeby, --goerli instead) (default: 1)
```